graphql_api/types/bundle_analysis_comparison/bundle_analysis_comparison.graphql

```diff
@@ -0,0 +1,24 @@
+union BundleAnalysisComparisonResult =
+    BundleAnalysisComparison
+  | FirstPullRequest
+  | MissingBaseCommit
+  | MissingHeadCommit
```

Comment:
Can this ever be `MissingHeadCommit`?. The function the resolver calls in `commit.py` does not return that type.

Reply:
It also gets called in `pull.py` which can return `MissingHeadCommit`